test/transcoding/SPV_INTEL_fpga_dsp_control/prefer_soft_logic.ll

```diff
@@ -0,0 +1,99 @@
+; template<typename Function>
```

Comment:
From the translator point of view, this and `prefer_soft_logic_propagate.ll` tests are not needed, because they were covered by `prefer_dsp_propagate.ll`: the input for the translator is LLVM IR and it is the same for all those tests, i.e. you don't check anything new. I suggest to remove those two files

If you want to check that particular DPC++ attribute was lowered into particular LLVM IR/SPIR-V, then it should be done in some other test in another project